src/schema/v2/notifications/__tests__/index.test.ts

```diff
@@ -165,7 +166,7 @@ describe("notificationsConnection", () => {
       it("should return `Today` label", async () => {
         const loader = () => {
           return Promise.resolve({
-            feed: [{ ...notificationFeedItem, date: moment() }],
+            feed: [{ ...notificationFeedItem, date: moment().tz(DEFAULT_TZ) }],
```

Comment:
_I think_ this is optional because `moment()` and `moment().tz(DEFAULT_TZ)` are the same point in time, just represent it in different timezone.